packages/office-ui-fabric-react/src/components/FocusZone/FocusZone.tsx

```diff
@@ -361,21 +367,18 @@ export class FocusZone extends BaseComponent<IFocusZoneProps, {}> implements IFo
           return;
 
         case KeyCodes.tab:
-          if (this.props.allowTabKey) {
+          if (this.props.tabPermission === FocusZoneTabbableElements.all
+            || (this.props.tabPermission === FocusZoneTabbableElements.inputOnly && this._isElementInput(ev.target as HTMLElement))) {
```

Comment:
put the || at the end of the previous line, it makes the if statement more readable